WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -0,0 +1,188 @@
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.ActionBarActivity;
+import android.view.MenuItem;
+import android.view.WindowManager;
+
+import com.simperium.client.BucketObjectMissingException;
+
+import org.wordpress.android.GCMIntentService;
+import org.wordpress.android.R;
+import org.wordpress.android.analytics.AnalyticsTracker;
+import org.wordpress.android.models.CommentStatus;
+import org.wordpress.android.models.Note;
+import org.wordpress.android.ui.WPWebViewActivity;
+import org.wordpress.android.ui.comments.CommentActions;
+import org.wordpress.android.ui.comments.CommentDetailActivity;
+import org.wordpress.android.ui.comments.CommentDetailFragment;
+import org.wordpress.android.ui.notifications.utils.SimperiumUtils;
+import org.wordpress.android.ui.reader.ReaderActivityLauncher;
+import org.wordpress.android.ui.reader.ReaderPostDetailFragment;
+import org.wordpress.android.ui.stats.StatsActivity;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.StringUtils;
+import org.wordpress.android.util.ToastUtils;
+
+public class NotificationsDetailActivity extends ActionBarActivity implements
+        CommentActions.OnNoteCommentActionListener {
+    private static final String ARG_TITLE = "activityTitle";
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        AppLog.i(AppLog.T.NOTIFS, "Creating NotificationsDetailActivity");
+
+        setContentView(R.layout.notifications_detail_activity);
+
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+
+        if (savedInstanceState == null) {
+            String noteId = getIntent().getStringExtra(NotificationsListFragment.NOTE_ID_EXTRA);
+            if (noteId == null) {
+                ToastUtils.showToast(this, R.string.error_notification_open);
+                return;
+            }
+
+            if (SimperiumUtils.getNotesBucket() != null) {
+                try {
+                    Note note = SimperiumUtils.getNotesBucket().get(noteId);
+
+                    // mark the note as read if it's unread
+                    if (note.isUnread()) {
+                        // mark as read which syncs with simperium
+                        note.markAsRead();
+                    }
+
+                    Fragment detailFragment = getDetailFragmentForNote(note);
+                    getFragmentManager().beginTransaction()
+                            .add(R.id.notifications_detail_container, detailFragment)
+                            .commit();
+
+                    if (getSupportActionBar() != null) {
+                        getSupportActionBar().setTitle(note.getTitle());
+                    }
+
+                    AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATIONS_ACCESSED);
```

Comment:
I noticed we're also tracking NOTIFICATIONS_ACCESSED [here](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2248-notifications-fragment/WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsActivity.java#L37). Shouldn't only one of these exist?